.github/workflows/ci.yml

```diff
@@ -148,7 +148,18 @@ jobs:
         run: mvn ${{ env.MAVEN_CLI_OPTS }} clean install -DskipTests -Dalfresco-platform.version=${{ matrix.repoVersion }}
       - name: "Build docker images"
         run: bash ./scripts/ci/buildDockerImages.sh
+      - name: "Start docker-compose and run e2e docker-compose tests"
+        env:
+          GHA_RUN_DC: true
+        run: |
+          ./scripts/ci/start-compose.sh distribution/src/main/resources/docker-compose/docker-compose-minimal.yml
+          mvn ${{ env.MAVEN_CLI_OPTS }} test -pl 'prediction-applier-extension,e2e-test' -am -Dtest=CreateNodeDCE2eTest -DfailIfNoTests=false -Dalfresco-platform.version=${{ matrix.repoVersion }}
```

Comment:
Can we make this more general with a wildcard? `*DCE2eTest` or something like that?

Reply:
We can, but will we need it to be more general?

Reply:
Currently only this test is suffering from this issue, so we don't need to generalise. When we add more d-c e2e tests then we can revisit this.